jobstore-persistence/src/test/java/uk/gov/moj/cpp/jobstore/persistence/AnsiSQLJobDataSqlProviderTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.moj.cpp.jobstore.persistence;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class AnsiSQLJobDataSqlProviderTest {
```

Comment:
We've moved everything  in framework to postgres only. I think this it the current default. In that case you won't need to implement an h2 version. Just make travis start a postgres instance when running the build

Reply:
A job for later, as discussed